tasks/34_max-triplet-sum/max-triplet-sum.js

```diff
@@ -0,0 +1,15 @@
+export function maxTripletSum(arr) {
```

Comment:
As I understood this, request can be summarised to:
- Filter out all duplicates
- Sort max to min
- Slice first three elements to create new array
- Sum up all the elements of this newly created array
Right?

Reply:
This can be done more readable way.

Hint is hidden in previous comment.

Reply:
Thanks to your summarization I made the code much more readable.